feat(docs): update product docs for self-driving IA (#18605)
PostHog/posthog.com: contents/docs/support/surfaces/slack.mdx

---
title: Use Support in Slack
sidebar: Docs
showTitle: true
---

<CalloutBox icon="IconFlask" title="Open beta" type="action">

The `@PostHog` Slack app is in beta, so some behaviors may still change.

</CalloutBox>

> Mention `@PostHog` in any Slack channel to query your support data and the [Self-driving](/docs/self-driving) work it generates. This is Slack as a surface, where you review and steer. It's separate from [SupportHog](/docs/support/slack), which is Slack as a channel that turns messages into tickets.

Reviewer: ```suggestion
Mention `@PostHog` in any Slack channel to query your support data and the [Self-driving](/docs/self-driving) work it generates. This is Slack as a surface and let's you do work by chatting with an agent in Slack. It's separate from [SupportHog](/docs/support/slack), which is Slack as a channel that turns messages into tickets.
```

This page covers using `@PostHog` for support. For everything the app can do across the rest of PostHog, see the [Slack app docs](/docs/slack).

## What you can do

Mention `@PostHog` and it works in the thread:

- Ask about your support data, like `summarize my open tickets` or `what's the oldest unresolved one?`
- Review the [Self-driving reports](/docs/self-driving/reports) built from your conversations, and approve, steer, or decline them.
- Ask it to plan a fix. It runs checks and opens a draft pull request, replying in-thread as it goes.

## Why Slack

> Two things make Slack useful for a support team specifically.

Reviewer: A third is there might already be context in the Slack thread, which the PostHog Agent sees
